<launch>
    <arg name="output_screen" default="screen"/>

    <!--BLUE ROBOT-->
    <node pkg="p_g06_camera_and_lidar" name="blue_camera_and_lidar" type="camera_and_lidar.py" output="$(arg output_screen)">
        <param name="robot_color" value="Blue"/>
        <remap from="blue1/cmd_vel" to="blue1/cmd_vel"/>
        <remap from="blue1/camera/rgb/image_raw" to="blue1/camera/rgb/image_raw"/>
        <remap from="blue1/scan" to="blue1/scan"/>
    </node>

</launch>
